card/lib/card/loader.rb

```diff
@@ -31,7 +31,7 @@ def load_chunks
       end
 
       def load_layouts
-        mod_dirs.inject({}) do |hash, mod|
+        mod_dirs.each_with_object({}) do |mod, hash|
```

Comment:
as the name suggests: first the 'each' then the 'object' :)  inject arguments are reversed